src/main/resources/application.conf

```diff
@@ -144,50 +147,60 @@ mantis {
     }
 
     rpc {
-      # JSON-RPC mode
-      # Available modes are: http, https
-      # Choosing https requires creating a certificate and setting up 'certificate-keystore-path' and
-      # 'certificate-password-file'
-      # See: https://github.com/input-output-hk/mantis/wiki/Creating-self-signed-certificate-for-using-JSON-RPC-with-HTTPS
-      mode = "http"
-
-      # Whether to enable JSON-RPC endpoint
-      enabled = true
-
-      # Listening address of JSON-RPC HTTP/HTTPS endpoint
-      interface = "localhost"
-
-      # Listening port of JSON-RPC HTTP/HTTPS endpoint
-      port = 8546
-
-      # Path to the keystore storing the certificates (used only for https)
-      # null value indicates HTTPS is not being used
-      certificate-keystore-path = null
+      http {
+        # JSON-RPC mode
+        # Available modes are: http, https
+        # Choosing https requires creating a certificate and setting up 'certificate-keystore-path' and
+        # 'certificate-password-file'
+        # See: https://github.com/input-output-hk/mantis/wiki/Creating-self-signed-certificate-for-using-JSON-RPC-with-HTTPS
+        mode = "http"
+
+        # Whether to enable JSON-RPC endpoint
```

Comment:
Should this be
> Whether to enable JSON-RPC HTTP(S) endpoint
?

Also, minor, we could use the shorter `HTTP(S)` instead of  `HTTP/HTTPS`